<launch>
    <param name="acc_noise"        type="double" value="1e-2" />
    <param name="gyr_noise"        type="double" value="1e-4" />
    <param name="acc_bias_noise"   type="double" value="1e-6" />
    <param name="gyr_bias_noise"   type="double" value="1e-8" />

    <param name="gps_f_x_imuFrame"   type="double" value="1.47" />
    <param name="gps_f_y_imuFrame"   type="double" value="0" />
    <param name="gps_f_z_imuFrame"   type="double" value="0" />

    <param name="gps_b_x_imuFrame"   type="double" value="0" />
    <param name="gps_b_y_imuFrame"   type="double" value="0" />
    <param name="gps_b_z_imuFrame"   type="double" value="0" />
  

    <param name="acc_std_threshold" type="double" value="3" />
    <param name="heading_std_threshold" type="double" value="3" />
    <param name="delta_gps_threshold" type="double" value="0.01" />

    <!-- UNIST -->
    <param name="local_long"       type="double" value="35.571127" />
    <param name="local_lat"        type="double" value="129.1874088" />
    <param name="local_alt"        type="double" value="76.716" />
    <!-- Daugu PG -->
    <!-- <param name="local_long"       type="double" value="35.6475476" />
    <param name="local_lat"        type="double" value="128.4026479" />
    <param name="local_alt"        type="double" value="48.827" /> -->


    <param name="frame_id"         type="string" value="map_gnss" />

    <param name="pub_tf"           type="bool" value="true" />
    <param name="child_frame_id"   type="string" value="base_link" />

    <param name="topic_imu"         type="string" value="/imu/data" />
    <param name="topic_gps_front"   type="string" value="/gnss_2/llh_position" />
    <param name="topic_gps_behind"  type="string" value="/gnss_1/llh_position" />

    <param name="viz_path"         type="bool" value="true" />
    <!-- <node name="nmea_topic_driver" pkg="nmea_navsat_driver" type="nmea_topic_driver" output="screen" /> -->
    
    <node name="imu_gnss_fusion_node" pkg="imu_gnss_fusion" type="imu_gnss_fusion_dual_node" output="screen"/>

    <node pkg="rviz" type="rviz" name="rviz" output="screen" 
      args="-d $(find imu_gnss_fusion)/rviz/default.rviz" required="true">
    </node>
</launch>